programs/code-of-conduct.md

```diff
@@ -0,0 +1,11 @@
+## Code of Conduct
```

Comment:
I don't understand why the CoC is being repeated in a subdirectory when it is already prominently linked to elsewhere and is in the root of the repo?

Reply:
I thought i removed this file in #909, but it appears to still be here. Will double check/correct on Monday.